modules/sdk-coin-sui/test/unit/keyPair.ts

```diff
@@ -0,0 +1,61 @@
+import assert from 'assert';
+import { KeyPair } from '../../src';
+import utils from '../../src/lib/utils';
+import should from 'should';
+import { Ed25519BIP32, Eddsa, HDTree } from '@bitgo/sdk-core';
+
+describe('SUI KeyPair', function () {
```

Comment:
could we add a test case where someone creates a `SuiKeyPair` w/ pub using an on chain address?

i.e.
```
new KeyPair({ pub: '0x...' });
```

Reply:
Still digging into seeing if there's a way to derive a pub key from a sui address. So far we have support to accept a hex/base58 format public key which is what is currently used within WP when trying to retrieve an address.

Reply:
yeah i didn't realize that the sui address is a hash of the public key, which makes it a 1 way operation. ignore this.